lib/gatherable/configuration.rb

```diff
@@ -20,8 +20,22 @@ def schema_name
       @schema_name || 'gatherable'
     end
 
-    def auth_method
-      @auth_method
+    def prefixed_resources
+      @prefixed_resources ||= []
+    end
+
+    def prefixed_resources=(resources)
```

Comment:
what happens if someone puts this at the top of their configuration block? Just demand that people put it at the bottom?